src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -204,6 +297,65 @@ std::array<tt::remove_cvref_wrap_t<T>, Dim> Translation<Dim>::coord_helper(
   return result;
 }
 
+template <size_t Dim>
+template <typename T>
+std::array<tt::remove_cvref_wrap_t<T>, Dim> Translation<Dim>::inspiral_helper(
+    const std::array<T, Dim>& source_coords, double time,
+    const std::unordered_map<
+        std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+        functions_of_time,
+    const size_t function_or_deriv_index) const {
+  ASSERT(functions_of_time.count(f_of_t_name_) == 1,
+         "The function of time '" << f_of_t_name_
+                                  << "' is not one of the known functions of "
+                                     "time. The known functions of time are: "
+                                  << keys_of(functions_of_time));
```

Comment:
We don't need to add this anymore. I made some changes that does this check one level above in the actual coordinate map.

Reply:
okay, should I take this out of the other spots in the translation as well?

Reply:
If there are any other places that check if the function of time name is in the functions of time, yes. But I think this is the only one?

Reply:
oh whoops yeah, I think I'm remembering from when the coord helper had it